java/pom.xml

```diff
@@ -0,0 +1,181 @@
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>2.10.4</version>
+                <configuration>
+                    <excludePackageNames>*.internal.*</excludePackageNames>
+                    <failOnError>false</failOnError>
+                    <additionalparam>-Xdoclint:none</additionalparam>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                        <inherited>true</inherited>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.6</version>
+                <configuration>
+                    <useDefaultManifestFile>true</useDefaultManifestFile>
+                    <archive>
+                        <index>true</index>
+                        <manifest>
+                            <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+                        </manifest>
+                    </archive>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.20</version>
+                <configuration>
+                    <surefire.useFile>false</surefire.useFile>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-release-plugin</artifactId>
+                <version>2.5.3</version><!--$NO-MVN-MAN-VER$-->
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <name>Google Diff Match and Patch</name>
+    <url>https://github.com/google/diff-match-patch</url>
+    <description>Diff Match Patch is a high-performance library in multiple languages that manipulates plain text.</description>
+    <inceptionYear>XXXX</inceptionYear>
+    <organization>
+        <url>http://www.google.com</url>
+        <name>Google Inc</name>
+    </organization>
+    <scm>
+        <url>https://github.com/google/diff-match-patch</url>
+        <connection>scm:git:git@github.com:google/diff-match-patch</connection>
+        <developerConnection>scm:git:git@github.com:google/diff-match-patch</developerConnection>
+      <tag>google-analytics-java-1.1.2</tag>
+  </scm>
```

Comment:
Indentation seems wrong here.  Also, the rest of this file is a random mix of 2, 3, and 4 space indentations.

Reply:
Will take care of it.